pyls/rpc_manager.py

```diff
@@ -118,7 +117,7 @@ def _handle_request(self, request):
         except JSONRPCDispatchException as e:
             output = _make_response(request, error=e.error._data)
         except Exception as e:  # pylint: disable=broad-except
-            log.error('endpoint exception %s %s %s', e.__class__.__name__, e.args, str(e))
+            log.exception('synchronous method handler exception')
```

Comment:
Can we log the original `request` here like:
```python
log.exception('synchronous method handler exception for request %s', request)
```

Reply:
sure! in a follow up

Reply:
https://github.com/palantir/python-language-server/issues/264